### DISCUSSION TITLE
Buyer persona y publico objetivo

### DISCUSSION THREAD

**Discussion by jenniferguerreroc** [Launch-X-Latam/MisionFrontEnd#125] [Q&A] [answered]
Hola! agradecería que me saquen de esta duda:
El **buyer persona** sería la persona que me solicita la pagina web? 
Y en cuanto al **publico objetivo**, serian las personas a las que va destinada esa pagina?

**Comment by ghost** [comment]
Yo creo que  asi es.

**Comment by gjordanra** [comment]
si no me equivoco, al igual que en marketing, el publico objetivo es el rango de personas hacia quienes va dirigido el sitio web que estas creando, ejemplo si es una tienda de ropa, el ambito de edades, sexo, ingresos y ubicación entre otros de el publico y la buyer persona es que tu creas unas cuantas personas imaginarias de ese publico, ejemplo tu dices, de ese publico, le voy a poner Alonso, Alonso vive en, su edad es de, se define como, le gusta ir al gym, la ropa de marca, los perros, quiere tener una mejor casa, es soltero y ahora defines otra, ejemplo Laura, Laura vive en, le gusta ..., tiene un ingreso de tanto y asi con unos cuantos para darte una idea de como serian las personas dentro de ese publico meta.

**Reply by jenniferguerreroc** [reply]
Muchas gracias!

**Comment by GeovannyCh** [comment, marked as answer]
**El buyer persona no es el que te pide la web**, este es un concepto del marketing utilizado para **imaginar al comprador ideal** del producto o servicio que estés ofreciendo como marca/empresa.
Tanto el buyer persona como el publico objetivo, **son una forma de segmentación del mercado**. por lo que entiendo, con el buyer persona se hace una s**egmentación mas precisa y detallada** ya que **buscamos gustos y patrones de comportamiento** en común entre nuestros posibles clientes, mientras que con la segmentación por publico objetivo, solo buscamos **rasgos de forma general** (edad, ubicación, genero, etc).

Te dejo un par de enlaces si quieres saber mas:

https://www.iebschool.com/blog/guia-crear-buyer-persona-inbound-marketing/

https://rockcontent.com/es/blog/publico-objetivo/

**Reply by jenniferguerreroc** [reply]
Muchas gracias!! Me quedo más que claro!
